gmplot/gmplot.py

```diff
@@ -169,9 +177,12 @@ def heatmap(self, lats, lngs, threshold=10, radius=10, gradient=None, opacity=0.
         settings['dissipating'] = dissipating
         settings = self._process_heatmap_kwargs(settings)
 
+        if weights is None:
```

Comment:
probably `if not weights`? if people pass `[]` (ok that'd be weird) the `zip()` call later on will return no results

maybe we should also introduce checks that `len(lats) == len(lngs) == len(weights)` ?

Reply:
Hmm, I was actually trying to avoid a case where people pass in `0`...

I think we can leave that as is, but yea we could replace all `zip()` calls with a wrapper function that ensures the length of all `zip()` arguments are equal to each other. If that works for you, I can create a new issue and implement this wrapper function there.